lib/core/src/lib/search-text/search-text-input.component.ts

```diff
@@ -99,6 +99,11 @@ export class SearchTextInputComponent implements OnInit, OnDestroy {
     @Input()
     placeholder: string = '';
 
+
+    /** Label text to show over the input field */
+    @Input()
+    label: string = '';
```

Comment:
I guess the [documentation](https://github.com/Alfresco/alfresco-ng2-components/blob/ACS-7973-acc-header-layout/docs/core/components/search-text-input.component.md) for search-text-input.component.ts should also be updated with this new input property